src/ocean/io/select/protocol/fiber/FiberSelectWriter.d

```diff
@@ -43,6 +43,8 @@ import ocean.stdc.posix.netinet.in_: IPPROTO_TCP;
 
 static if (__VERSION__ >= 2000 && __VERSION__ < 2073)
     enum { TCP_CORK = 3 }
+else static if (__VERSION__ >= 2077)
+    import core.sys.linux.netinet.tcp: TCP_CORK;
```

Comment:
An explanation on why `core.sys.linux.netinet.tcp` can't be used or the time (or `core.sys.linux.sys.netinet.tcp`) would be nice

Reply:
Not sure if it's worth it, but maybe the commit comment can go in the code too.